src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -38,4 +39,9 @@ public Driver update(Driver driver) {
     public boolean delete(Long id) {
         return driverDao.delete(id);
     }
+
```

Comment:
same here with creating new package  service/impl and move impl class to impl package